source/partialObject.js

```diff
@@ -29,5 +29,7 @@ import _curry2 from './internal/_curry2.js';
  *      sayHelloToMs({ firstName: 'Jane', lastName: 'Jones' }); //=> 'Hello, Ms. Jane Jones!'
  * @symb R.partialObject(f, { a, b })({ c, d }) = f({ a, b, c, d })
  */
-var partialObject = _curry2((f, o) => (props) => f.call(this, mergeDeepRight(o, props)));
+var partialObject = _curry2(function partialObject(f, o) {
+  return (props) => f.call(this, mergeDeepRight(o, props));
```

Comment:
Let's convert this arrow function and return `function _partialObject(props) { ... }` here as well. For now ramda is not supposed to be using any arrow functions in `source`.